aat/src/java/uk/gov/hmcts/ccd/caseactivity/befta/CaseActivityTestAutomationAdapter.java

```diff
@@ -8,9 +10,14 @@
     private TestDataLoaderToDefinitionStore loader = new TestDataLoaderToDefinitionStore(this);
 
     @Override
-    public void doLoadTestData() {
-        loader.addCcdRoles();
-        loader.importDefinitions();
+    protected BeftaTestDataLoader buildTestDataLoader() {
+        return new DefaultBeftaTestDataLoader() {
+            @Override
+            public void doLoadTestData() {
+                CaseActivityTestAutomationAdapter.this.loader.addCcdRoles();
+                CaseActivityTestAutomationAdapter.this.loader.importDefinitions();
+            }            
+        };
```

Comment:
@VamshiRudrabhatlaHMCTS  - This is the bit you need to apply in all test suites to apply the breaking change coming with befta 6.0. :)